cmd/validator/validator.go

```diff
@@ -65,21 +66,46 @@ func validatorUsage() {
 	flag.PrintDefaults()
 }
 
+// Assemble pretty formatted list of file types
+func getFileTypes() []string {
+	options := make([]string, 0, len(filetype.FileTypes))
+	for _, typ := range filetype.FileTypes {
+		options = append(options, typ.Name)
+	}
+	return options
+}
+
+// Given a slice of strings, validates each is a valid file type
+func validateFileTypeList(input []string) bool {
+	types := getFileTypes()
+	for _, t := range input {
+		if len(t) == 0 {
+			continue
+		}
+		if !slices.Contains(types, t) {
+			return false
+		}
+	}
+	return true
+}
+
 // Parses, validates, and returns the flags
 // flag.String returns a pointer
 // If a required parameter is missing the help
 // output will be displayed and the function
 // will return with exit = 1
 func getFlags() (validatorConfig, error) {
 	flag.Usage = validatorUsage
-	depthPtr := flag.Int("depth", 0, "Depth of recursion for the provided search paths. Set depth to 0 to disable recursive path traversal")
-	excludeDirsPtr := flag.String("exclude-dirs", "", "Subdirectories to exclude when searching for configuration files")
-	excludeFileTypesPtr := flag.String("exclude-file-types", "", "A comma separated list of file types to ignore")
-	outputPtr := flag.String("output", "", "Destination to a file to output results")
-	reportTypePtr := flag.String("reporter", "standard", "Format of the printed report. Options are standard and json")
-	versionPtr := flag.Bool("version", false, "Version prints the release version of validator")
-	groupOutputPtr := flag.String("groupby", "", "Group output by filetype, directory, pass-fail. Supported for Standard and JSON reports")
-	quietPrt := flag.Bool("quiet", false, "If quiet flag is set. It doesn't print any output to stdout.")
+	var (
+		depthPtr            = flag.Int("depth", 0, "Depth of recursion for the provided search paths. Set depth to 0 to disable recursive path traversal")
+		excludeDirsPtr      = flag.String("exclude-dirs", "", "Subdirectories to exclude when searching for configuration files")
+		excludeFileTypesPtr = flag.String("exclude-file-types", "", "A comma separated list of file types to ignore.\nValid options: "+strings.Join(getFileTypes(), ","))
```

Comment:
This is real picky but can you add a space after the commas so the list looks like this:

```
optional flags:
  -depth int
    	Depth of recursion for the provided search paths. Set depth to 0 to disable recursive path traversal
  -exclude-dirs string
    	Subdirectories to exclude when searching for configuration files
  -exclude-file-types string
    	A comma separated list of file types to ignore.
    	Valid options: json, yaml, xml, toml, ini, properties, hcl, plist, csv, hocon, env, editorconfig
```

rather than this:

```
optional flags:
  -depth int
    	Depth of recursion for the provided search paths. Set depth to 0 to disable recursive path traversal
  -exclude-dirs string
    	Subdirectories to exclude when searching for configuration files
  -exclude-file-types string
    	A comma separated list of file types to ignore.
    	Valid options: json,yaml,xml,toml,ini,properties,hcl,plist,csv,hocon,env,editorconfig
```

Would also like the list sorted alphabetically if possible